agent/agentssh/jetbrainstrack.go

```diff
@@ -0,0 +1,85 @@
+package agentssh
+
+import (
+	"strings"
+	"sync"
+
+	"cdr.dev/slog"
+	"github.com/gliderlabs/ssh"
+	"go.uber.org/atomic"
+	gossh "golang.org/x/crypto/ssh"
+)
+
+type localForwardChannelData struct {
+	DestAddr string
+	DestPort uint32
+
+	OriginAddr string
+	OriginPort uint32
+}
+
+type ChannelAcceptWatcher struct {
```

Comment:
Lets call it something pretty direct?
```suggestion
type JetbrainsNewChannel struct {
```